crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2074,8 +2068,15 @@ impl SymbolicationActor {
                 ..
             } = result;
 
-            if let Some(client_stacktraces) = client_stacktraces {
-                merge_clientside_with_processed_stacktraces(&mut stacktraces, client_stacktraces);
+            match client_stacktraces {
+                Ok(Some(client_stacktraces)) => merge_clientside_with_processed_stacktraces(
+                    &mut stacktraces,
+                    client_stacktraces,
+                ),
+                Err(e) => {
+                    log::error!("invalid minidump extension: {}", e);
+                }
+                _ => {}
```

Comment:
do we still have a way to track stack traces that breakpad isn't able to parse with this change? is that useful or meaningful info for us?